pkg/inngest/inngest/connect/_internal/init_handshake_handler.py

```diff
@@ -25,13 +25,15 @@ def __init__(
         app_configs: dict[str, list[server_lib.FunctionConfig]],
         env: str | None,
         instance_id: str,
+        max_worker_concurrency: int | None = None,
```

Comment:
```suggestion
        max_worker_concurrency: int | None,
```

Removing the default ensures we never forget to pass it

Reply:
Updated